css/style.css

```diff
@@ -287,6 +287,11 @@ pre code {
 	background-color: #fcf8e3;
 	border-color: #faebcc;
 }
+.alert-danger {
+	color: #a94442;
+	background-color: #f2dede;
+	border-color: #ebccd1;
+}
```

Comment:
This is a simple copy-paste of Bootstrap defaults, just like previous definition (`.alert-warning`). I'm not planning on making this more religiously clean as other improvements of this will take over during doc improvements.